src/command_modules/azure-cli-webapp/azure/cli/command_modules/webapp/_help.py

```diff
@@ -12,9 +12,94 @@
     short-summary: commands for managing your Azure web apps, app service plans, etc
 """
 
-helps['appservice plan'] = """
+helps['appservice web config'] = """
     type: group
-    short-summary: commands for managing your app service plans
+    short-summary: commands to configure the web
```

Comment:
You know, the command category is `web`, I thought I could use it with less words, but with your comments and checked out more msdn doc, like [this one](https://azure.microsoft.com/en-us/documentation/articles/app-service-changes-existing-services/?clcid=0x409) I think we should use `web app`